gui/src-tauri/tauri.conf.json

```diff
@@ -78,7 +78,7 @@
         "decorations": false,
         "transparent": false,
         "fileDropEnabled": false,
-        "visible": true
+        "visible": false
```

Comment:
```suggestion
        "visible": true
```
This needs to be true or else the window is invisible .-.

Reply:
That’s not what visible means, and weird I tested it in my side

Reply:
The window is invisible if false and visible if true tho…

Reply:
Nvm, after further testing, ig that doesn't matter.
This still doesn't fix the window bug for me tho